docs/deployment.md

```diff
@@ -0,0 +1,204 @@
+# OpenSRP Deployment
+
+### Setting up an opensrp web instance
+
+We support 3 deployment environments: stage, preview and production so one should first decide under which environment they want to set up the site in.
+
+You also need to have the necessary SSH keys set up in your machine in order to gain access to the deployment server. Kindly refer to [this](https://github.com/onaio/infrastructure/blob/35172d3e0986d4f55749a780122326340b40e35a/docs/tools/ansible_playbooks.md) doc on how to generate and sign the SSH keys.
```

Comment:
@manutarus @bennsimon Do we give this access to the community? Our aim is to have demo or test ansible deployment scripts for the community to use.

Reply:
@dubdabasoduba @KipSigei who is the planned audience of this content, and in the worst case how  restricted can their access be to Artifacts involved in the process described?